### ISSUE TITLE
Buried Pages in Development

### ISSUE THREAD

**Issue by rgleason** [opencpn-manuals/user#3]
@rgleason : This also includes some files marked as KEEP , aimed for the user manual. They were blocking #3, and was out of scope anyway. HOWEVER, before I removed them i tagged the sources with the tag rick-um-keep i. e., if you do git checkout git-um-keep the files are available again.

Which files please?

**Comment by rgleason** [comment]
https://github.com/opencpn-manuals/development/issues/30

**Comment by leamas** [comment]
Actually, to have things in the git history doesn't mean they are buried by any stretch of the imagination. This is just  about some basic git skills; I have tried to explain in https://github.com/opencpn-manuals/main/discussions/14#discussioncomment-1199498.

TL;DR: The following checks out all files as they were before they was deleted:

    $ git pull --tags origin
    $ git checkout rick-um-keep

Buried?
